svd-rs/src/clusterinfo.rs

```diff
@@ -146,7 +147,14 @@ impl ClusterInfo {
     pub fn builder() -> ClusterInfoBuilder {
         ClusterInfoBuilder::default()
     }
-
+    /// Construct single `Cluster`
```

Comment:
This, and the other instances, should maybe be an intra-doc link, feels kinda redundant but I usually do it anyway myself

Reply:
square brackets?